docs/api/en/math/Color.html

```diff
@@ -91,13 +133,13 @@ <h3>[property:Boolean isColor]</h3>
 		<p>Read-only flag to check if a given object is of type [name].</p>
 
 		<h3>[property:Float r]</h3>
-		<p>Red channel value between `0` and `1`. Default is `1`.</p>
+		<p>Red channel value, typically between `0` and `1`. Default is `1`.</p>
```

Comment:
I do not like the use of "typically". That just raises more questions than it answers. For now, I'd leave it as [0, 1].

We can figure out how to address out-of-gamut values later.